### ISSUE TITLE
WiFi icon is not working on POI card

### ISSUE THREAD

**Issue by mlbiche** [organicmaps/organicmaps#9656]
**Describe the issue**
When selecting a POI that proposes WiFi, the icon in the card subtitle is not working.

**Steps to reproduce**
1. Select a POI that proposes WiFi (I.e. “Overland Greece GuestHouse” in Greece > Thessalie - Central Greece card : https://osm.org/go/xxZGXCAf-?m).
2. See the WiFi icon being a question mark in a grey square.

**Expected behaviour**
Would except the proper WiFi icon.

**Screenshots**
![013DD8EC-A8A0-4A77-9DF5-25A2A114AD11](https://github.com/user-attachments/assets/e10704ea-d7c1-4c79-93c3-33218dbd9c72)

**System information:**
 - Operating system and its version: iOS15.8.3
 - Organic Maps version: 2024.11.12-6
 - Device Model: iPhone SE 1st generation

**Comment by map-per** [comment]
Related to the Android issue #9640

**Comment by matheusgomesms** [comment]
This heart emoji (working fine before), disappeared as well. Perhaps, it's the same issue? (iOS here).

![IMG_4801](https://github.com/user-attachments/assets/a74a21cc-b394-4537-9424-748971427760)

**Comment by biodranik** [comment]
@matheusgomesms a heart is displayed on your screenshot. When is it not visible? On which iOS version?

@kirylkaveryn do you have iOS 15 to test disappeared wifi icon? Why did it disappear? Did we change the font used for the subtitle?

**Comment by matheusgomesms** [comment]
> a heart is displayed on your screenshot. When is it not visible? On which iOS version?

Right on the map. PP page is fine, but map no. I remember (might be wrong here) that before it didn't happen. iOS 17.6.1 and latest AppStore OM version.

**Comment by kirylkaveryn** [comment]
> @matheusgomesms a heart is displayed on your screenshot. When is it not visible? On which iOS version?
> 
> @kirylkaveryn do you have iOS 15 to test disappeared wifi icon? Why did it disappear? Did we change the font used for the subtitle?

@biodranik I don't have ios 15... only 12.5 and >17

**Comment by batterydotac** [comment]
FWIW, I see the same crosshatch pattern instead of a heart like in the screenshot of @matheusgomesms on my LG G Pad 8.3 running Android 5.0.2 and also on my Pixel 7 running Android 15.  Both show the heart correctly in the Place Page title.  However, the original issue by @mlbiche doesn't seem to be present on either device as the WiFi icon appears properly.  So these are probably separate concerns.  I'm using a very recent copy of OM (compiled from source) as of this posting date.

**Comment by biodranik** [comment]
Non-rendered emoji on the map is a separate issue from the Place Page.

Android Place Page issues are only indirectly related to iOS in that some system fonts are missing. 

@kirylkaveryn which font is specified for the PP subtitle? Can you repro the same issue on iOS 12?
